components/calculator/misdemeanor/QandAContainer.tsx

```diff
@@ -1,6 +1,7 @@
 import {
   Box, Button, Container, Stack,
 } from '@mui/material';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Can you remove this line? It shouldn't be needed with the package being in the package.json file. @shondriane